HW3/app/src/main/java/com/example/hw3/NotificationReceiver.java

```diff
@@ -0,0 +1,96 @@
+package com.example.hw3;
+
+import android.annotation.SuppressLint;
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+
+import androidx.core.app.NotificationCompat;
+import androidx.core.app.NotificationManagerCompat;
+
+import java.util.Random;
+
+public class NotificationReceiver extends BroadcastReceiver {
+    private static final String CHANNEL_ID = "notificationChannel";
+    protected int mNotificationId = 1;
+
+    private final String[] quotes = {

```

Comment:
Why not creating a new Quotes class that holds this objects so you will have more OOP structure to your code? (: